eos-live-boot-overlayfs-setup

```diff
@@ -47,7 +49,7 @@ setup_ostree_flatpak_overlay() {
 
 # Everything but /sysroot/{ostree,flatpak} is pretty straightforward:
 overlay_dirs="bin boot endless home lib opt ostree root sbin srv sysroot/home var"
-for dir in $overlay_dirs; do
+for dir in $overlay_dirs ${EXTRA_PATHS[@]}; do
```

Comment:
I don't think `$EXTRA_PATHS` is actually an array as defined here, so the `${EXTRA_PATHS[@]}` syntax has no effect:

```console
$ x=(a b); for y in ${x[@]}; do echo $y; done
a
b
$ cat x
#!/bin/bash
EXTRA_PATHS=$@
for dir in ${EXTRA_PATHS[@]}; do echo $dir; done
$ ./x "a b"
a
b
```

I think you want either:

```bash
EXTRA_PATHS=( "$@" )
for dir in $overlay_dirs "${EXTRA_PATHS[@]}"; do
```

or in the likely case that we really do not care about paths with spaces in them:

```bash
EXTRA_PATHS=$@
for dir in $overlay_dirs $EXTRA_PATHS
```